File: qe-java-selenium/Activity1.java
package demos;

import org.openqa.selenium.By;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity1 {
    public static void main(String[] args) throws InterruptedException {
        FirefoxDriver driver = new FirefoxDriver();
        driver.get("https://training-support.net");

        System.out.println(driver.getTitle());

        driver.findElement(By.linkText("About Us")).click();

        Thread.sleep(1000);

        System.out.println(driver.getTitle());

        driver.quit();
    }

}

File: qe-selenium-python/Activity1.py
from selenium import webdriver
from selenium.webdriver.common.by import By
import time


with webdriver.Firefox() as driver:
    driver.get("https://training-support.net")

    print (driver.title)

    driver.find_element(By.LINK_TEXT, "About Us").click()

    time.sleep(1)

    print(driver.title)


    driver.quit()